code/game/objects/items/theft_tools.dm

```diff
@@ -203,6 +203,7 @@
 	toolspeed = 0.5
 	damtype = "fire"
 	usesound = 'sound/weapons/bladeslice.ogg'
+	var/usesLeft = 2
```

Comment:
Shouldn't this be slightly random to prevent meta?

Reply:
yeah do 2-4